static/css/subject_page.scss

```diff
@@ -15,7 +15,7 @@
  */
 
 /**
- * Styling for disaster dashboard.
+ * Styling for sustainability explorer.
```

Comment:
i think this should be for subject pages? but also, wondering if we should actually keep disaster dashboard and sustainability explorer files separate on the client side since the two routes are still different on the flask server and since they're technically independent apps?